src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -9380,23 +9374,29 @@ .maxstack  2
                     C V_18,
                     object V_19,
                     int V_20)
-     -IL_0000:  nop
-     ~IL_0001:  newobj     ""C.<>c__DisplayClass0_0..ctor()""
+      // sequence point: {
+      IL_0000:  nop
+      // sequence point: <hidden>
```

Comment:
Why does the closure class creation get a sequence point? What purpose does that serve in debugging?

Reply:
I believe these will be necessary to enable edit-and-continue for switch expressions. In any case, that didn't change here.  We are just displaying the information in a different format (sequence point comments instead of funny squiggles before the instructions due to the change at line 9353).

---
In reply to: [416899613](https://github.com/dotnet/roslyn/pull/43471#discussion_r416899613) [](ancestors = 416899613)